inc/Engine/Cache/PurgeActionsSubscriber.php

```diff
@@ -88,6 +89,16 @@ public function purge_user_cache( $user_id ) {
 	 * @return void
 	 */
 	public function maybe_purge_cache_on_term_change( $term_id, $tt_id, $taxonomy ) {
+		/**
+		 * Filter use to determine if we are currently importing data into the WordPress.
+		 * Bails out if this filter returns true.
+		 *
+		 * @param boolean Tells if we are importing or not.
+		 */
+		if ( (bool) apply_filters( 'rocket_importing_mode', false ) || defined( 'WP_IMPORTING' ) ) {
+			return;
+		}
```

Comment:
The filter is defined 6 times, do u think it'd be better having a dedicated function for it so it can be maintained easily?